.github/workflows/build-test.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python: ["3.8"]
+        python: ["3.11"]
```

Comment:
I don't believe this is relevant to anything since each creates its own conda environment. However, I'm bumping it on general principles.